drv/stm32h7-spi-server-core/Cargo.toml

```diff
@@ -32,5 +32,14 @@ build-util = { path = "../../build/util" }
 call_rustfmt = { path = "../../build/call_rustfmt" }
 
 [features]
+# These features are used to select a global SPI peripheral block; making this
+# choice based on features prevents extra rebuilds when building a complete
```

Comment:
Personally I'd make this longer by adding something like

`These features are *not* referenced anywhere in the code, they exist only at build time, but should not be removed without measuring impact on build times.`

Reply:
Yup, I updated this to clarify that the features modify code-generation (since we're using them to select the global block), but aren't used by name anywhere in the code.